packages/fxa-js-client/client/lib/hawkCredentials.js

```diff
@@ -1,37 +1,38 @@
+'use strict';
+
```

Comment:
No biggie, but it seems weird that `'use strict'` has gone in above the MPL preamble comment for these. It seems more conventional to put the preamble first (and that's what we do everywhere else).